src/main/java/org/cactoos/scalar/And.java

```diff
@@ -39,10 +39,13 @@
  * {@link java.util.stream.Stream#forEach(java.util.function.Consumer)}
  * works:</p>
  *
- * <pre> new And(
- *   new IterableOf("Mary", "John", "William", "Napkin"),
- *   name -> System.out.printf("The name: %s\n", name)
- * ).value();</pre>
+ * <pre>
+ * new And(
```

Comment:
@neonailol what about indentation? I see in some places the indentation is of 2 spaces, in some of 4. I think we should keep it consistend, of 4 spaces.